docs/proposals/configmap.md

```diff
@@ -293,12 +296,12 @@ spec:
         - name: config-map-volume
           mountPath: /mnt/config-map
   volumes:
-  - name: config-map-volume
-    configMap:
-      files:
-        - path: "etc/redis.conf"
-          configMapName: redis-volume-config
-          key: redis.conf
+    - name: config-map-volume
+      configMap:
+        configMapName: redis-volume-config
```

Comment:
SecretVolumeSource.SecretName is `omitempty` already, so you don't NEED to default it, strictly.  I still would.

If we're going to move this way, why add ConfigMapName at all, if we just intend to deprecate it?